test/test-normalize-hotkey.js

```diff
@@ -0,0 +1,36 @@
+import {normalizeHotkey} from '../dist/index.js'
+
+describe('normalizeHotkey', () => {
+  it('should exist', () => {
+    assert.isDefined(normalizeHotkey)
+  })
+
+  const tests = [
+    // Base case control tests
+    ['a', 'a'],
+    ['Control+a', 'Control+a'],
+    ['Meta+a', 'Meta+a'],
+    ['Control+Meta+a', 'Control+Meta+a'],
+    // Mod should be localized based on platform
+    ['Mod+a', 'Control+a', 'win'],
+    ['Mod+a', 'Meta+a', 'mac'],
+    ['Mod+A', 'Control+A', 'win'],
+    ['Mod+A', 'Meta+A', 'mac'], // TODO: on a mac upper-case keys are lowercased when Meta is pressed
+    ['Mod+9', 'Control+9', 'win'],
+    ['Mod+9', 'Meta+9', 'mac'],
+    ['Mod+)', 'Control+)', 'win'],
+    ['Mod+)', 'Meta+)', 'mac'], // TODO: on a mac upper-case keys are lowercased when Meta is pressed
+    ['Mod+Alt+a', 'Control+Alt+a', 'win'],
+    ['Mod+Alt+a', 'Alt+Meta+a', 'mac'],
+    // Modifier sorting
+    ['Shift+Alt+Meta+Control+m', 'Control+Alt+Meta+Shift+m'],
```

Comment:
I know this test is only meant to demonstrate sorting works but wondering if we should exclude this given it's invalid. Or maybe we can add a note that this is invalid [along with note to not use `Mod` with `Meta` or `Control`](https://github.com/github/hotkey/pull/66/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R82)

Reply:
I think it is valid to have shortcuts that pair `Control` and `Meta` (just not either of those keys with `Mod`).

Reply:
You're right, it's technically valid on mac!